### ISSUE TITLE
Cannot assign array to property verbb\giftvoucher\models\Settings::$codeStorage of type string

### ISSUE THREAD

**Issue by CMeldgaard** [verbb/gift-voucher#98]
### Describe the bug

When changing codestorage, as per the docs the system returns the error:
`Cannot assign array to property verbb\giftvoucher\models\Settings::$codeStorage of type string`

Using this in the gift-voucher.php
`<?php

use verbb\giftvoucher\storage\Order;

return [
    '*' => [
        'expiry' => 36,
        'codeKeyLength' => 10,
        'codeKeyCharacters' => 'ACDEFGHJKLMNPQRTUVWXYZ234679',
        'voucherCodesPdfPath' => 'shop/_pdf/voucher',
        'voucherCodesPdfFilenameFormat' => 'Voucher-{number}',
        'pdfAllowRemoteImages' => false,
        'pdfPaperSize' => 'A4',
        'pdfPaperOrientation' => 'portrait',
        'registerAdjuster' => 'beforeTax',
        'attachPdfToEmails' => [],
        'codeStorage' => ['class' => Order::class, 'fieldHandle' => 'codes']
    ]
];`

### Steps to reproduce

1. Follow the docs: https://verbb.io/craft-plugins/gift-voucher/docs/feature-tour/code-storage
2. Load the CP, and the error will be triggered

### Craft CMS version

4.2.1.1

### Plugin version

3.0.0

### Multi-site?

No

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/gift-voucher` requirement in `composer.json` to:

```
"require": {
  "verbb/gift-voucher": "dev-craft-4 as 3.0.0",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [3.0.1](https://github.com/verbb/gift-voucher/releases/tag/3.0.1)
